specification/cognitiveservices/data-plane/ContentSafety/preview/2023-04-30-preview/contentsafety.json

```diff
@@ -942,6 +942,27 @@
         "blocklistName"
       ]
     },
+    "TextBlocklistCreateOrUpdate": {
+      "type": "object",
+      "description": "Text Blocklist.",
+      "properties": {
+        "blocklistName": {
```

Comment:
This change seems semantically meaningful, since the `blocklistName` property was added.  Is this fixing a bug in the old generated autorest?

Reply:
Yes, definitely expected.